sparepo/build.py

```diff
@@ -306,12 +306,12 @@ def create_hashtable(
         )
 
         for file_number, counts in enumerate(counts_by_file):
-            output_count_array[:, file_number] = counts[part_type][:]
+            output_count_array[:, file_number] = counts[part_type.value][:]
```

Comment:
Again, this should be left as the enum.

The design is:

{
  Enum: {...}
}

in the dictionaries. We just need to use the integers when writing out to keep things consistent with GADGET

Reply:
This also need to be undo